logger/src/metrics.rs

```diff
@@ -389,6 +389,15 @@ pub struct MemoryMetrics {
     pub dirty_pages: SharedMetric,
 }
 
+/// Metrics related to signals.
```

Comment:
These metrics don't look useful because their value will always be 0 or 1. You can get this information from the exit status of Firecracker as we already use different exit codes for SIGBUS and SIGSEGV.

Any reason to add these metrics other than testing the incremented metric in unit tests?

Reply:
`seccomp.num_faults` is also a 0/1 metric and it's also used the same way, *here*. In production, it gets aggregated to show multiple seccomp failures. These signal-related metrics would do the same.